src/main/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperations.scala

```diff
@@ -408,6 +410,11 @@ object TezosDatabaseOperations extends LazyLogging {
     sql"""SELECT #$cols FROM #$table WHERE true """
   }
 
+  def makeOrdering(ordering: List[QueryOrdering]): SQLActionBuilder = {
```

Comment:
The more gets added to this class, the more I have the feeling that it would be better extracted to a general database module/class that builds generic queries on top of Slick.
There is a lot of logic now that is not tied to tezos specifically, and could be more aptly moved to DatabaseUtil, or to a similar QueryBuilder class, that would keep coupled things together

Reply:
You are right, I'll refactor this in the new PR with ticket I've got about adding limiting amount of rows returned